### PR TITLE
Update decorator description.

### PR DESCRIPTION
**Pull request by algmyr**
Add description of what a decorator is actually syntactic sugar for.

### CODE REVIEW

File: bot/resources/tags/decorators.md
Comment (shtlrs) [inline bot/resources/tags/decorators.md:2]: The branch is a bit out of date, can you rebase onto main/merge this please ?

**Review by shtlrs — CHANGES_REQUESTED** [review]

**Comment by Xithrius** [comment]
@algmyr Greetings, will you be continuing this PR?

Thanks!

**Comment by Xithrius** [comment]
@algmyr Hey, would you like to continue this PR? I can put it up for grabs, otherwise.

Thanks!

**Comment by Xithrius** [comment]
Author hasn't responded to 2 pings. I'm taking over this PR.

File: bot/resources/tags/decorators.md
Comment (Xithrius) [reply bot/resources/tags/decorators.md]: Done

**Comment by Xithrius** [comment]
I think the current example is enough for the tag. Also, there was no issue assigned, so next time please get approval from a core developer before proceeding with a PR.

Thanks!

**Pull request closed without merging**
